examples/simple_window/src/main.rs

```diff
@@ -0,0 +1,59 @@
+#![windows_subsystem = "windows"]
+
+use std::convert::TryFrom;
+
+use project_reunion_rs::{PackageVersion, workarounds};
+
+use bindings::{
+    Microsoft,
+    Microsoft::UI::Xaml::{
+        Application, ApplicationInitializationCallback, Controls::Button, HorizontalAlignment,
+        LaunchActivatedEventArgs, RoutedEventHandler, Window,
+    },
+};
+
+use windows::{implement, IInspectable, Interface, HRESULT};
+
+#[implement(extend Microsoft::UI::Xaml::Application, override OnLaunched)]
+struct App {
+    window: Option<Window>,
+}
+
+#[allow(non_snake_case)]
+impl App {
+    fn OnLaunched(&mut self, _: &Option<LaunchActivatedEventArgs>) -> windows::Result<()> {
+        self.window = Some(Window::new()?);
+        if let Some(window) = &self.window {
```

Comment:
Perhaps prefer to `unwrap` the window as it will always succeed and can then avoid the dead branch below.